monai/engines/workflow.py

```diff
@@ -87,6 +90,12 @@ def __init__(
         self.data_loader = data_loader
         self.prepare_batch = prepare_batch
 
+        if post_transform is not None:
+
+            @self.on(Events.ITERATION_COMPLETED)
+            def run_post_transform(engine):
+                engine.state.output = post_transform(engine.state.output)
```

Comment:
could use the `apply_transform` interface for better exception handling? 
https://github.com/Project-MONAI/MONAI/blob/da800411c71b3ddf092e070e5950e715f41c1c16/monai/transforms/utils.py#L240

Reply:
Hi @wyli ,

Thanks for your review.
I updated according to your comments.